CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ and this project adheres to
   Clarified what CPU models are supported by each existing CPU template.
   Firecracker exits with an error if a CPU template is used on an unsupported
   CPU model.
+- [#5165](https://github.com/firecracker-microvm/firecracker/pull/5165):
+  Removed Firecracker snapshot feature from developer preview status.
```

Comment:
could we say that we promoted them from DP to GA instead?

Reply:
Reworded. Does it look better?